buffer/src/test/java/io/netty/buffer/AbstractByteBufTest.java

```diff
@@ -2281,10 +2281,12 @@ public void run() {
                         byte[] array = new byte[8];
                         buf.readBytes(array);
 
+                        //noinspection MisorderedAssertEqualsArguments
```

Comment:
I would prefer not to add comments for specific IDEs in the code if possible. https://github.com/netty/netty/pull/6494#discussion_r104469753 it is not clear to me why this warning is triggered based upon the description ... neither of the arguments to the `assert` call are a literal ... do you have a better description of this warning than I provided in https://github.com/netty/netty/pull/6494#discussion_r104469753?